index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css"/>
+    </head>
+    <body>
+
```

Comment:
There are missing elements in the head section. Please add <meta> elements to head section.

Reply:
- [ ] Kindly highlight properly the area you are calling the attention of the student to and state the reasons for your request. This would give the student more insight of why you ask him or her to do a certain thing.

Reply:
Thanks for this review. Can you kindly indicate to me how I edit highlighting? I know how to edit the comment but I can't edit the highlighting. Should I write a new comment?